infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/HasErrorInformation.java

```diff
@@ -0,0 +1,8 @@
+package com.redhat.service.smartevents.infra.exceptions;
+
+public interface HasErrorInformation {
```

Comment:
@manstis Do we need this now you've moved the fields onto the `ManagedResource`?

Reply:
Technically you are correct; however it'd mean moving the error handling code from `infra` to `manager`. This, however, cannot be done as the error handling code used by both the `manager` and `operator.` The interface allows decoupling of these modules.